File: gluecodium/src/test/resources/smoke/duplicate_names/output/android/com/example/smoke/LearnToRead.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class LearnToRead {
    @NonNull
    public com.example.smoke.Alphabet fieldA;
    @NonNull
    public com.example.smoke.foo.Alphabet fieldB;
    public LearnToRead() {
        this.fieldA = com.example.smoke.Alphabet.A;
        this.fieldB = com.example.smoke.foo.Alphabet.BETA;
    }
}

File: gluecodium/src/test/resources/smoke/duplicate_names/output/android-kotlin/com/example/smoke/LearnToRead.kt
/*

 *
 */

@file:JvmName("LearnToRead")

package com.example.smoke


class LearnToRead {
    @JvmField var fieldA: com.example.smoke.Alphabet
    @JvmField var fieldB: com.example.smoke.foo.Alphabet



    constructor() {
        this.fieldA = com.example.smoke.Alphabet.A
        this.fieldB = com.example.smoke.foo.Alphabet.BETA
    }





}

